odbc/src/api/col_attr_describe_tests.rs

```diff
@@ -424,6 +424,7 @@ mod unit {
                         numeric_attr_ptr,
                     )
                 );
+                dbg!(desc);
```

Comment:
Remaining dbg! ;)

Reply:
No, that was is intentional :D It shows us where the test fails iff it fails.